_pages/docs/matching-options.md

```diff
@@ -74,6 +74,90 @@ Here's an example of match configuration options.
 </options>
 ```
 
+And here are the same options in JSON format:
+
+```json
+{
+  "options": {
+    "property-defs": {
+      "property": [
+        { "namespace": "", "localname": "IdentificationID", "name": "ssn" },
+        { "namespace": "", "localname": "PersonGivenName", "name": "first-name" },
+        { "namespace": "", "localname": "PersonSurName", "name": "last-name" },
+        { "namespace": "", "localname": "AddressPrivateMailboxText", "name": "addr1" },
+        { "namespace": "", "localname": "LocationCity", "name": "city" },
+        { "namespace": "", "localname": "LocationState", "name": "state" },
+        { "namespace": "", "localname": "LocationPostalCode", "name": "zip" }
+      ]
+    },
+    "algorithms": {
+      "algorithm": [
+        { "name": "std-reduce", "function": "standard-reduction" },
+        { "name": "std-reduce-query", "function": "standard-reduction-query" },
+        { "name": "dbl-metaphone", "function": "double-metaphone" }
+      ]
+    },
+    "scoring": [
+      {
+        "add": [
+          { "property-name": "ssn", "weight": "50" },
+          { "property-name": "last-name", "weight": "8" },
+          { "property-name": "first-name", "weight": "12" },
+          { "property-name": "addr1", "weight": "5" },
+          { "property-name": "city", "weight": "3" },
+          { "property-name": "state", "weight": "1" },
+          { "property-name": "zip", "weight": "3" }
+        ],
+        "expand": [
+          {
+            "property-name": "first-name",
+            "algorithm-ref": "thesaurus",
```

Comment:
probably too late for this comment... should we do camelCase?

Reply:
actually I see camel case in some places. weird.

Reply:
I agree, should be camel case